### ISSUE TITLE
SteamOS 3.0 please include KDE Connect

### ISSUE THREAD

**Issue by derpelikan** [ValveSoftware/SteamOS#749]
Please include KDE Connect into the SteamOS delivery. This tool would help integrating the SteamDeck with other appliances in the network, exchanging files and automating things in a secure way.
KDE Connect can not be put into a Flatpak nor AppImage, so the only way for users to get it would be inclusion in the distribution.

**Comment by Daasin** [comment]
This is definitely something i've been hoping for

**Comment by GodisGoodAlways** [comment]
True using led they should have led connect

**Comment by nogeenkopjethee** [comment]
> There's a Flatpak repo of KDE applications (here)[https://community.kde.org/Guidelines_and_HOWTOs/Flatpak].

**Comment by JLP** [comment]
Oh yes please. This is one of the things I miss the most on Steam Deck out of the box, apart from a video player like SMPlayer and a music player like Elisa. It would be even more awesome if the KDE COnnect functionality was integrated into game mode GUI so it would not be available only in desktop mode. For example sending the screenshot you make easy to a phone/computer in your home network.

**Comment by sevmonster** [comment]
KDE Connect *is* available as a flatpak. You just need to add the KDE repo. It works fine but you have to launch it and keep it open while using it, and some features don't work/aren't included like notification area integration. Also file transfers don't work, I think something to do with flatpak permissions and sandboxing; I'm sure there's some way past that but I used scp/sshfs instead of bothering with it.

**Comment by revanmj** [comment]
I guess the idea is that if they included it, they would also fix those issues with file transfers, so people wouldn't have to deal with them manually.

**Comment by sevmonster** [comment]
I'm not disagreeing with packaging it natively, just giving some context to @derpelikan's comments.

**Comment by sevmonster** [comment]
Oh by the way I got file transfers to work. I had to add `~/deck/Downloads:rw` flatpak file permission, it uses that folder by default.
Changing the directory through the UI doesn't work, but after attempting to change it (the field remains visually blank) it you can close Connect and manually modify the config files in the flatpak directory, and it works after adding the file permission for the new target folder.

**Comment by Daasin** [comment]
And if adding KDE Connect; a similar linked issue would be not being able to change firewall settings from the KDE Desktop Settings due to the prerequisites only being partially implemented. Why not just get the full thing and make it a whole distro? 🤷🏻‍♂️ 🤔

**Comment by Atem18** [comment]
https://store.steampowered.com/news/app/1675200/view/3383920059429575957

Tl;dr: 5.26 is coming as well as kdeconnect

**Comment by JLP** [comment]
Really awesome news and update.

**Comment by Atem18** [comment]
Yes this should be closed as KDE Connect is working well, I even tested it from my macbook and iPhone.

**Comment by sevmonster** [comment]
@kisak-valve You can probably close this issue. KDE Connect is natively packaged and works fine.

**Comment by derpelikan** [comment]
Resolution confirmed
